components/Account.php

```diff
@@ -158,6 +158,7 @@ public function onRegister()
          */
         if ($userActivation) {
             $this->sendActivationEmail($user);
+            Flash::success(trans('rainlab.user::lang.account.activation_email_sent'));
```

Comment:
Added missing flash-message if user activation is set to true at the backend settings.